components/header.tsx

```diff
@@ -18,7 +26,24 @@ export default function Header() {
           </a>
         </Link>
       </Flex>
-      <Flex as="nav" sx={{ justifyContent: "center" }}>
+      <Flex
+        as="nav"
+        className={isActive ? "d-flex" : ''}
```

Comment:
instead of using a class you can have the same in the sx prop. 
But I'm a bit confused here. This component should have display flex already by default, why adding a class when active? Is it to override the display none when is in the mobile mediaquery?

So I bet you can just do:
```
{
            display: isActive ? 'flex' : 'none',
}
```
in the following media query.
In general do you see the power of having just javascript? You really don't need css classes here